### PR TITLE
Storage STG 82

### PR DESCRIPTION
**Pull request by seanmcc-msft**

### CODE REVIEW

**Comment by JeffreyRichter** [comment]
Why are all these breaking changes shown [here](https://github.com/Azure/azure-rest-api-specs/pull/18074/checks?check_run_id=5412359997)?

**Comment by seanmcc-msft** [comment]
> Why are all these breaking changes shown [here](https://github.com/Azure/azure-rest-api-specs/pull/18074/checks?check_run_id=5412359997)?

@tg-msft

This is the same issue we ran into with https://github.com/Azure/azure-rest-api-specs/pull/18069.

File: specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-10-02/blob.json
Comment (tg-msft) [inline specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-10-02/blob.json:7724]: How do customers opt-in to this behavior?  Is it only returned when marker or maxresults is provided?

This might end up causing problems with code gen - in which case maybe we need to split this apart into separate operations (one legacy "here's everything" and a new pageable alternative).

**Review by tg-msft — APPROVED** [review]
All of those breaking changes are because of an ancient DataLake swagger file.  We should fix this by copying all the files used for Track 2 swagger from "Microsoft.\*Storage" to "Azure.Storage.\*" folders so the tooling stops complaining.  We're trying to unblock other work today though so we'll do that in the future.  I'm going to add the "breaking changes approved" label since these aren't actual breaking changes.

The blobs changes are easy to read in the PR, but you can check out [this diff for Files](https://portal.azure-devex-tools.com/tools/diff/eyJzb3VyY2VVcmwiOiJodHRwczovL2dpdGh1Yi5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvYmxvYi9iOTVkMmE1ZDg5ZTQwNDI3MjBiYmYwMTk5YmMyYjY3NDhkZTlkODlmL3NwZWNpZmljYXRpb24vc3RvcmFnZS9kYXRhLXBsYW5lL01pY3Jvc29mdC5GaWxlU3RvcmFnZS9wcmV2aWV3LzIwMjEtMDQtMTAvZmlsZS5qc29uIiwidGFyZ2V0VXJsIjoiaHR0cHM6Ly9naXRodWIuY29tL0F6dXJlL2F6dXJlLXJlc3QtYXBpLXNwZWNzL2Jsb2IvYjk1ZDJhNWQ4OWU0MDQyNzIwYmJmMDE5OWJjMmI2NzQ4ZGU5ZDg5Zi9zcGVjaWZpY2F0aW9uL3N0b3JhZ2UvZGF0YS1wbGFuZS9NaWNyb3NvZnQuRmlsZVN0b3JhZ2UvcHJldmlldy8yMDIxLTA2LTA4L2ZpbGUuanNvbiJ9) and [this diff for DataLake](https://portal.azure-devex-tools.com/tools/diff/eyJzb3VyY2VVcmwiOiJodHRwczovL2dpdGh1Yi5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvYmxvYi9iOTVkMmE1ZDg5ZTQwNDI3MjBiYmYwMTk5YmMyYjY3NDhkZTlkODlmL3NwZWNpZmljYXRpb24vc3RvcmFnZS9kYXRhLXBsYW5lL01pY3Jvc29mdC5TdG9yYWdlRGF0YUxha2UvcHJldmlldy8yMDIxLTA0LTEwL0RhdGFMYWtlU3RvcmFnZS5qc29uIiwidGFyZ2V0VXJsIjoiaHR0cHM6Ly9naXRodWIuY29tL0F6dXJlL2F6dXJlLXJlc3QtYXBpLXNwZWNzL2Jsb2IvYjk1ZDJhNWQ4OWU0MDQyNzIwYmJmMDE5OWJjMmI2NzQ4ZGU5ZDg5Zi9zcGVjaWZpY2F0aW9uL3N0b3JhZ2UvZGF0YS1wbGFuZS9NaWNyb3NvZnQuU3RvcmFnZURhdGFMYWtlL3ByZXZpZXcvMjAyMS0wNi0wOC9EYXRhTGFrZVN0b3JhZ2UuanNvbiJ9) to see what's being added _(note that site is kind of flaky and you may need to refresh several times before it finally loads anything)_.

File: specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-10-02/blob.json
Comment (seanmcc-msft) [reply specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-10-02/blob.json:7724]: That is correct.  If customers don't provide `maxresults` or `marker`, the service defaults to the old behavior.

I actually haven't had an issue in the .NET code gen, since these parameters are optional.

**Comment by tjprescott** [comment]
@seanmcc-msft I can't merge this with the Prettier check failing. Are those something you can address?

**Comment by seanmcc-msft** [comment]
> @seanmcc-msft I can't merge this with the Prettier check failing. Are those something you can address?

Yes, I think I have addressed this issue.

**Comment by seanmcc-msft** [comment]
@tjprescott, can we merge?

**Pull request merged**
